server/athenian/api/controllers/reposet_controller.py

```diff
@@ -97,6 +98,22 @@
         return e.response
     rss = await request.sdb.fetch_all(
         select([RepositorySet]).where(RepositorySet.owner == id))
+    if len(rss) == 0:
+        # new account, discover the repos the metadata DB
+        async with request.mdb.connection() as conn:
+            iid = await conn.fetch_val(
+                select([InstallationOwner.install_id])
+                .where(InstallationOwner.user_id == "user_id_from_Auth0"))
```

Comment:
Isn't the id taken from the `request`?

Reply:
This is exactly the place where I am blocked: this requires knowing the exact format of the JWT token which, from @dennwc's words, contains the GitHub ID.

There is a (crappy) workaround to query the mgmt API - it returns the nice `identities` list.

Reply:
okay, sorry for dealying this, asap I'll ping you when integrating webapp <> api.